src/components/SecondaryNav/_secondary-nav.scss

```diff
@@ -0,0 +1,15 @@
+.p-secondary-nav {
+  .p-list__link {
+    color: #111;
```

Comment:
I'm aware this should be a Vanilla variable but they cannot currently be accessed. Issue raised: https://github.com/canonical-web-and-design/jaas-dashboard/issues/13

Reply:
In maas-ui and the old gui we import the Vanilla bits we need at the top of the individual files, in this case that would be `@import "node_modules/vanilla-framework/scss/settings_colors";`

Reply:
I've submitted a PR here: https://github.com/canonical-web-and-design/jaas-dashboard/pull/23

Managed to simplify the import to `"@import "vanilla-framework/scss/vanilla";"` which can be used uniformly in all partials and allow all Vanilla entities to be referenced. As Vanilla is engineering so all patterns are wrapped in mixins, only what is referenced is included so no performance overhead.